core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -101,7 +101,7 @@
 			),
 			'toggle' => array
 			(
-				'href'                => 'act=toggle&amp;field=invisible',
```

Comment:
I would expect a `toggleField => 'invisible'` or something like thtat still. Otherwise, if you add a second toggle field to the same DCA, you would also have to adjust the `href` of the previous toggle operation. That makes no sense imho.

Reply:
if you have multiple fields, you need to tell what field to toggle. I thought if you only have one field you might not want to add that to the config?